src/dashboard/Data/Browser/BrowserTable.react.js

```diff
@@ -21,7 +21,7 @@ import PropTypes              from 'lib/PropTypes';
 
 const MAX_ROWS = 200; // Number of rows to render at any time
 const ROWS_OFFSET = 160;
-const ROW_HEIGHT = 31;
+const ROW_HEIGHT = 30;
```

Comment:
Is it intetional?

Reply:
I rebased on master, should be gone now.